share/completions/diskutil.fish

```diff
@@ -8,131 +12,188 @@ function __fish_diskutil_mounted_volumes
+complete -f -c diskutil -n '__fish_diskutil_using_not_subcommand splitPartition' -a '(__fish_diskutil_devices)'
 
 # mergePartitions
 complete -f -c diskutil -n '__fish_use_subcommand' -a mergePartitions -d 'Merge two or more partitions on a disk'
-complete -f -c diskutil -n '__fish_seen_subcommand_from mergePartitions' -a '(__fish_diskutil_devices)'
+complete -f -c diskutil -n '__fish_diskutil_using_not_subcommand mergePartitions' -a '(__fish_diskutil_devices)'
+
+# apfs
+complete -f -c diskutil -n '__fish_use_subcommand' -a apfs -d 'Merge two or more partitions on a disk'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a list -d 'Show status of all current APFS Containers'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a convert -d 'Nondestructively convert from HFS to APFS'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a create -d 'Create a new APFS Container with one APFS Volume'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a createContainer -d 'Create a new empty APFS Container'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a deleteContainer -d 'Delete an APFS Container and reformat disks to HFS'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a resizeContainer -d 'Resize an APFS Container and its disk space usage'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a addVolume -d 'Export a new APFS Volume from an APFS Container'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a deleteVolume -d 'Remove an APFS Volume from its APFS Container'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a eraseVolume -d 'Erase contents of, but keep, an APFS Volume'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a changeVolumeRole -d 'Change the Role metadata bits of an APFS Volume'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a unlockVolume -d 'Unlock an encrypted APFS Volume which is locked'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a lockVolume -d 'Lock an encrypted APFS Volume (diskutil unmount)'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a listCryotoUsers -d 'List cryptographic users of encrypted APFS Volume'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a setPassphraseHint -d 'Change the passphrase of a cryptographic user'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a encryptVolume -d 'Set or clear passphrase hint of a cryptographic user'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a decryptVolume -d 'Start async encryption of an unencrypted APFS Volume'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a updatePreboot -d 'Start async decryption of an encrypted APFS Volume'
+complete -f -c diskutil -n '__fish_seen_subcommand_from apfs' -a changePassphrase -d 'Update the APFS Volume\'s related APFS Preboot Volume'
```

Comment:
Yes, these are shuffled - this is the description for "updatePreboot" (I guess).